src/packages/recompose/createEventHandler.js

```diff
@@ -0,0 +1,23 @@
+import $$observable from 'symbol-observable'
+import { createChangeEmitter } from 'change-emitter'
+import { getTransform } from './configureObservable'
+
+const createEventHandler = () => {
+  const emitter = createChangeEmitter()
+  const transform = getTransform()
```

Comment:
@chicoxyzzy Thanks for links. Also observer dont has interop point like `symbol-observable` - https://github.com/zenparsing/es-observable/issues/89. Perhaps my proposal was a bit hasty :)